images/__common.sh

```diff
@@ -24,10 +24,9 @@ function downloadFileAndVerifyChecksum() {
     rm -f $headersFile
     echo
     echo "Verifying checksum..."
-
     checksumcode="sha512sum"
     if [ "$platformName" == "golang" ];then
         checksumcode="sha256sum"
-   fi
+    fi
```

Comment:
nit: remove space

Reply:
@arroyc if I remove the space, the `fi` wont alight with the `if`